docs/release-notes/2.32.0.md

```diff
@@ -0,0 +1,156 @@
+
+## New Features
+
+- [PMM-9366](https://jira.percona.com/browse/PMM-9366): [Backup Management]: Added possibility to restore MongoDB PITR backups. This option is available under **Backup > All Backups > Actions > Restore this backup** for any PITR-type backups available in the table.
+- [PMM-9368](https://jira.percona.com/browse/PMM-9368): [Backup Management]: Added support for local client storage: When adding storage locations under **Backup > Storage Location**, you can now also select **Local Client** option as the location type for storing future backups.
+- [PMM-10599](https://jira.percona.com/browse/PMM-10599), [PMM-10503](https://jira.percona.com/browse/PMM-10599): [Backup Management]:  Refactored User Interface for creating backups to enable you to create on-demand and scheduled backups from the same window instead of using separate modals. 
+- [PMM-10470](https://jira.percona.com/browse/PMM-10470): [Backup Management]: Following the support for MongoDB physical backups introduced in PMM 2.31, we have now added the possibility to restore these backups.  Restoring MongoDBphysical backups requires additional manual operations after the restore and is only available with Percona Server for MongoDB.
+- [PMM-10502](https://jira.percona.com/browse/PMM-10502), [PMM-10831](https://jira.percona.com/browse/PMM-10831): [Backup Management]: Improved User Experience and labels on all four tabs of the Backup section: 
+    - Added icons to better illustrate failed and successful backups.
+    - Added **Service Name** column to show the service from where the backup was retrieved.
+    - Renamed **DB Vendor** tab to **DB Technology**.
+    - Improved **Actions** menu with clearer labeled options.
+
+- [PMM-10742](https://jira.percona.com/browse/PMM-10742): [Backup Management]: Extended PMM compatibility with Percona Backup for MongoDB (PMB) to support the latest version, v. 2.0 PBM.
+- [PMM-10829](https://jira.percona.com/browse/PMM-10829): [Backup Management]: The backups table under **Backup > All Backups** now includes a new **Location** column which displays the service where the backup came from (S3 or Local Storage).
+
+ 
+- [PMM-8792](https://jira.percona.com/browse/PMM-8792), [PMM-9055](https://jira.percona.com/browse/PMM-9055), [PMM-9056](https://jira.percona.com/browse/PMM-9056): PMM 2.32.0 introduces the parameter `--max-query-length` in the `pmm-admin` command for MYSQL, PostgreSQL, and MongoDB databases. It can be also setup through UI. Now, queries longer than 2048 characters are not truncated (if the parameter is set to `-1`, which means no limit or value greater than 2048 chars), and the **Explain** and **Examples** tabs on the QAN dashboard do not throw an error, see [PMM-1899](https://jira.percona.com/browse/PMM-1899). These tabs are important as they provide contextual information pertaining to the query.
```

Comment:
```suggestion
- [PMM-8792](https://jira.percona.com/browse/PMM-8792),  [PMM-9055](https://jira.percona.com/browse/PMM-9055), [PMM-9056](https://jira.percona.com/browse/PMM-9056) - PMM 2.32.0 introduces the parameter --max-query-length in the pmm-admin command for MYSQL, PostreSQL, and MongoDB databases. It can be also setup through UI during adding of service. You can see new input in form, where you can setup you value of max query length. With these changes, queries longer than 2048 characters are not truncated (if parameter is set -1, which means no limit, or value is bigger than 2048), and the Explain and Examples tabs on the QAN dashboard do not throw an error ([PMM-1899](https://jira.percona.com/browse/PMM-1899)). These tabs provide contextual information pertaining to the query, thus enhancing user experience.
```